asserts/crypto.go

```diff
@@ -34,3 +42,129 @@ func generatePrivateKey() (*packet.PrivateKey, error) {
 	}
 	return packet.NewRSAPrivateKey(time.Now(), priv), nil
 }
+
+var openpgpConfig = &packet.Config{
+	DefaultHash: crypto.SHA256,
+}
+
+func signContent(content []byte, privKey *packet.PrivateKey) ([]byte, error) {
+	sig := new(packet.Signature)
+	sig.PubKeyAlgo = privKey.PubKeyAlgo
+	sig.Hash = openpgpConfig.Hash()
+	sig.CreationTime = time.Now()
+	sig.IssuerKeyId = &privKey.KeyId
+
+	h := openpgpConfig.Hash().New()
+	h.Write(content)
+
+	err := sig.Sign(h, privKey, openpgpConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	buf := bytes.NewBufferString("openpgp ")
+	enc := base64.NewEncoder(base64.StdEncoding, buf)
```

Comment:
I'm fine with this going in like this for now, but on a follow up iteration can we please introduce appropriate line breaks to have a nicely formatted text file?